examples/30_extended/study_tutorial.py

```diff
@@ -104,7 +103,7 @@ def cat(X):
     [("cat", cat_imp, cat), ("cont", FunctionTransformer(lambda x: x, validate=False), cont)]
 )
 clf = sklearn.pipeline.Pipeline(
-    steps=[("transform", ct), ("estimator", HistGradientBoostingClassifier()),]
+    steps=[("transform", ct), ("estimator", HistGradientBoostingClassifier())]
```

Comment:
Here we actually also have the imports in two places, could you please clean these as well?